system/logaccess/Grafana/CurlClient/GrafanaCurlClient.cpp

```diff
@@ -370,11 +412,20 @@ void GrafanaLogAccessCurlClient::processQueryJsonResp(LogQueryResultDetails & re
 
         bool isFirstLine = true;
         Owned<IPropertyTreeIterator> resultIter = data->getElements("result");
-        //many result elements can be returned, each with a unique set of labels
+        //many result elements can be returned, each with a unique set of labels and a common set of stream values
         ForEach(*resultIter)
         {
             IPropertyTree & result = resultIter->query();
-            wrapResult(returnbuf, &result, format, isFirstLine);
+            Owned<IPropertyTreeIterator> logLineIter;
+
+            if (result.hasProp("values"))
+            {
+                logLineIter.setown(result.getElements("values")); // if no values elements found, will get NullPTreeIterator
+                if (!logLineIter)
```

Comment:
This doesn't seem necessary

Reply:
k, the goal was to protect processValues from null logLineIter, removing in favor of L271